### ISSUE TITLE
Bitcom as OS

### ISSUE THREAD

**Issue by unwriter** [unwriter/Bitcom#1]
# Bitcom as OS

> Proposed by @torusJKL

Original Proposal on Bitcoin: https://viewer.bitdb.network/#e52864d04efa56d94b1a3cf6347a752ed4114349caf191ed6ea08f865643e085

Further explanation: https://twitter.com/_unwriter/status/1089517509437464578

Instead of thinking of Bitcom as a "global computer that all application protocols register to",
think of Bitcom as "an OS (operating system) that gets installed into protocols".

Let's say your OP_RETURN protocol for a game controller looked like this:

```
1. Up
  OP_RETURN 1Frooui3mqRQa1nAv8VRUDaRLxSXRFyNXA Up

2. Down
  OP_RETURN 1Frooui3mqRQa1nAv8VRUDaRLxSXRFyNXA Down

3. Left
  OP_RETURN 1Frooui3mqRQa1nAv8VRUDaRLxSXRFyNXA Left

4. Right
  OP_RETURN 1Frooui3mqRQa1nAv8VRUDaRLxSXRFyNXA Right

5. A Key
  OP_RETURN 1Frooui3mqRQa1nAv8VRUDaRLxSXRFyNXA A

6. B Key
  OP_RETURN 1Frooui3mqRQa1nAv8VRUDaRLxSXRFyNXA B
```

To "install Bitcom OS" to your protocol, you can "activate" the Bitcom OS by starting your
OP_RETURN with "$" to indicate the "Admin mode":

```
OP_RETURN 1Frooui3mqRQa1nAv8VRUDaRLxSXRFyNXA $ echo "Torus Game Controller" to description
```

Basically,

1. you didn't have to register to anywhere.
2. the "Bitcom" protocol is just a part of your protocol
